src/entities/Ticket.java

```diff
@@ -111,23 +112,23 @@ public void calculatePrice() {
 
     }
     
-    public int[] getPricesFromDB() {
-		int[] prices = null;
-		FileInputStream fis = null;
-		ObjectInputStream in = null;
-		
-		try {
-			fis = new FileInputStream(priceFileName);
-			in = new ObjectInputStream(fis);
-			prices = (int[]) in.readObject();
-			in.close();
-		} catch (IOException ex) {
-			ex.printStackTrace();
-		} catch (ClassNotFoundException ex) {
-			ex.printStackTrace();
-		}
-		return prices;
-	}
+//    public int[] getPricesFromDB() {
```

Comment:
Commented this out since we are not storing any persistent data for ticket anymore